presto-main/src/main/java/com/facebook/presto/type/MapType.java

```diff
@@ -42,6 +47,7 @@
 {
     private static final ObjectMapper OBJECT_MAPPER = new ObjectMapperProvider().get().registerModule(new SimpleModule().addSerializer(Slice.class, new SliceSerializer()));
     private static final ObjectMapper RAW_SLICE_OBJECT_MAPPER = new ObjectMapperProvider().get().registerModule(new SimpleModule().addSerializer(Slice.class, new RawSliceSerializer()));
+    private static final String mapCompareNullMsg = "Map comparison not supported for maps with null keys";
```

Comment:
Since this is a static final field name it `MAP_COMPARE_NULL_MESSAGE` also start the message with "MAP comparison..." to match the array message